gcp-pipelines/x86_64-linux-clang.yml

```diff
@@ -0,0 +1,41 @@
+steps:
+  - name: 'gcr.io/shaderc-build/shader-compiler-team:kokoro-dxc-builder'
+    args:
+      - git
+      - submodule
+      - update
+      - '--init'
+  - name: 'gcr.io/shaderc-build/shader-compiler-team:kokoro-dxc-builder'
+    args:
+      - cmake
+      - '-Bbuild'
+      - '-GNinja'
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-C'
+      - cmake/caches/PredefinedParams.cmake
+      - '-DSPIRV_BUILD_TESTS=ON'
+      - '-DCMAKE_C_COMPILER=clang'
+      - '-DCMAKE_CXX_COMPILER=clang++'
+      - '-DENABLE_SPIRV_CODEGEN=ON'
+  - name: 'gcr.io/shaderc-build/shader-compiler-team:kokoro-dxc-builder'
+    args:
+      - ninja
+      - '-C'
+      - build
+  - name: 'gcr.io/shaderc-build/shader-compiler-team:kokoro-dxc-builder'
+    args:
+      - ninja
+      - '-C'
+      - build
+      - check-all
+logsBucket: 'gs://public-github-building-logs'
+options:
+  logging: LEGACY
+  pool:
+    name: projects/shaderc-build/locations/europe-west1/workerPools/dxc
```

Comment:
The pool we have configured on GCP (to work with GCE and other network related stuff) is un `europe-west1`, but we could create pools in other regions. The motivation I see could be quotas, is there another reason?

Reply:
Latency for folks over this side of the pond, but this is a batch builder so that's not a big deal.  Let's start with this and change it in the future, if needed.